syntax = "proto3";
package grpc.example;

service GreetingService {
    rpc SubscribeGreetingBySecond (stream RequestSubscribe) returns (stream ReplyGreeting) {
    }
}

message RequestSubscribe {
    // request multi times to change name
    string name = 1;
    uint64 current_nanosecond = 2;
}

message ReplyGreeting {
    // reply: "name: count"
    string message = 1;
    uint64 current_nanosecond = 2;
}